CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## 2021-07-06, v1.15
+
+- Updated the generator phpdoc to help identifier magic methods (#307)
+- Prevent direct access and triggered deprecation warning for "word" me (#302)
```

Comment:
the `me` at the end looks weird

Reply:
Yeah correct, its a copy paste so will adjust